src/ui/registry/units/common/types/components/OAuthTokenButton.ts

```diff
@@ -0,0 +1,7 @@
+import type {ButtonProps} from '@gravity-ui/uikit';
+
+export type OAuthTokenButtonProps = ButtonProps & {
+    onTokenChange: (token: string) => void;
+    application: any;
```

Comment:
Why any? Maybe limit it to `string`?

Reply:
String does not satisfy real type of OAuthButton from closed part of our code. I will think about it, maybe there is a better approach